### DISCUSSION TITLE
Ethereum wallets intergation

### DISCUSSION THREAD

**Discussion by Belief208** [community/community#126666] [Programming Help] [answered]
### Body

What are the main types of Ethereum wallets, and how do they differ?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by degen130** [comment]
Hello cuteolef

I saw your post and then want to help you
The main types of Ethereum wallets are:
Physical devices that store private keys offline, providing a high level of security. Examples include Ledger and Trezor.
Applications or browser extensions that store private keys on the user's device. Examples include MetaMask (browser extension) and MyEtherWallet (web-based).
Smartphone applications that store private keys on the mobile device. Examples include Trust Wallet and Coinbase Wallet.
These are helpful for your project
Thank you in advance

**Comment by Belief208** [comment]
Wow!
Good!
This is great way!
Thank you

**Comment by degen130** [comment]
Thank you too

**Comment by degen130** [comment, marked as answer]
Have a good day
